(235) Rationalise all phone numbers

dgmstuart opened the pull request:
There are a lot of pages in the site which include phone numbers. In some
 cases the numbers were inconsistent, and how they were presented on the
 page was inconsistent as well.

 Now there is only one place where phone numbers are explicitly written, and
 everywhere they are used in the pages they are used in the same way: as a
 telephone number link with the appropriate 'itemprop'.

 Note that in some places the specific number used on a page has been
 corrected to the most appropriate one.

dgmstuart committed:
(refactor/fix) Rationalise all phone numbers

There are a lot of pages in the site which include phone numbers. In
some cases the numbers were inconsistent, and how they were presented on
the page was inconsistent as well.

Now there is only one place where phone numbers are explicitly written,
and everywhere they are used in the pages they are used in the same way:
as a telephone number link with the appropriate 'itemprop'.

Note that in some places the specific number used on a page has been
corrected to the most appropriate one.

dgmstuart commented:
rebased

leeky commented on spec/helpers/application_helper_spec.rb:
```diff
@@ -0,0 +1,10 @@
+  describe '#telephone_number' do
+    it 'wraps the phone number in appropriate markup' do
+      expect(helper.telephone_number('01234 567 890'))
+        .to eq(%(<span itemprop="telephone"><a href="tel:01234567890">01234567890</a></span>))
```
Shouldn't this test fail? The code is `ApplicationHelper` only removed the spaces from the `href` and not the string contained within - so `helper.telephone_number('01234 567 890')` should generate:

```html
... <a href="tel:01234567890">01234 567 890</a> ...
```

dgmstuart replied:
Good point! Will investigate

dgmstuart replied:
Ok, that was weird: I was (incorrectly) using `gsub!` instead of `gsub`, and I guess that gets evaluated first because in the markup href comes before the text.

Damn mutation!

dgmstuart committed:
(fix) preserve spaces in phone numbers

gsub! mutates the variable, and since the markup evaluates the href
before the link text it was stripping the spaces out of both.

dgmstuart commented:
@leeky Rebased and pushed a commit to fix

leeky commented:
LGTM! :shipit:
